src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -78,6 +80,15 @@
         // which port the I2C device plugged in on the multiplexer
         public static final int LEFT_INTAKE_COLORSENSOR_PORT = 1, RIGHT_INTAKE_COLORSENSOR_PORT = 2,
                 CENTER_INTAKE_COLORSENSOR_PORT = 3;
+
+        /*
+         * According to
+         * https://docs.wpilib.org/en/stable/docs/software/can-devices/power-distribution-module.html
+         * The CAN ID for CTRE should be 0
+         * Or 1 for Rev (we're using Rev for the new robot)
+         */
+        public static final int PDP_CAN_ID = 9;
```

Comment:
This isn't 0 or 1.  Do we use a non-default CAN ID for the power distribution panel?  If so, add a comment saying that so someone doesn't "fix" this and break it.